python/tank/bootstrap/manager.py

```diff
@@ -215,10 +202,10 @@ def _get_progress_callback(self):
 
             progress_callback(progress_value, message)
 
-        where:
-        - ``progress_value`` is the current progress value, a float number ranging from 0.0 to 1.0
-                             representing the percentage of work completed.
-        - ``message`` is the progress message string to report.
+            # progress_value is the current progress value, a float number
+            # ranging from 0.0 to 1.0 and representing the percentage of work completed.
+            #
+            # message is the progress message string to report.
```

Comment:
the sphinx docs were broken before, tried to make the bullet points work but it looked really odd. This is what it looks like now:

![image](https://cloud.githubusercontent.com/assets/337710/18587070/f6cb3c76-7c17-11e6-9fc9-16c649c8d455.png)

You want me to polish that further?